internal/experimental/registry/client.go

```diff
@@ -206,7 +213,12 @@ type (
 )
 
 // Pull downloads a chart from a registry
-func (c *Client) Pull(ref string, options ...PullOption) (*PullResult, error) {
+func (c *Client) Pull(rawRef string, options ...PullOption) (*PullResult, error) {
```

Comment:
In both `Pull` and `Push` the variable in the function signature is being changed to `rawRef` and `ref` is the parsed version. Would it be possible to keep the function sig the same and instead call the internal one `parsedRef`? Since the type is changing the use of this internally had to change.

Reply:
absolutely! I'm not married to the initial proposed naming, it's been updated in fed0d31